src/iris/vendors/iris_slack.py

```diff
@@ -125,8 +126,17 @@ def send_message(self, message):
                     return time.time() - start
                 # If message is invalid:
                 #   {u'ok': False, u'error': u'invalid_arg_name'}
-                logger.error('Received an error from slack api: %s',
-                             data['error'])
+
+                # Check the slack api to see if the iris bot is in the channel first before logging an error
+                # if not in the channel, the error is expected and should not be logged
+                msg_endpoint = self.config['base_url'] + "/conversations.info"
+                response_alt = requests.get(msg_endpoint, data=payload, proxies=self.proxy, timeout=self.timeout)
+                if response_alt.status_code == 200 \
+                        and response_alt.json()['ok'] \
+                        and not response_alt.json()['channel']['is_member']:
+                    pass
+                else:
```

Comment:
we probably still want to log at least an Info line so that we can easily see if there are users who are missing message deliveries